animation_nodes/nodes/vector/noise_utils.pyx

```diff
@@ -0,0 +1,75 @@
+from mathutils import Vector, noise
+from ... data_structures cimport (DoubleList, Vector3DList)
+
+#Blender Fractal Functions:
+
+def blFractal(str noiseBasis, Vector3DList vectors, float fractalDimension, float lacunarity ,int octaves):

```

Comment:
Change this `float lacunarity ,int octaves) --> float lacunarity, Py_ssize_t octaves)`.
Same for other functions.